.github/workflows/healthchecks_clickhouse_ci.yml

```
@@ -0,0 +1,76 @@
+name: HealthChecks ClickHouse DB CI
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ master ]
+    paths:
+      - src/HealthChecks.ClickHouse/**
+      - test/HealthChecks.ClickHouse.Tests/**
+      - test/_SHARED/**
+      - .github/workflows/healthchecks_clickhouse_ci.yml
+      - Directory.Build.props
+      - Directory.Build.targets
+    tags-ignore:
+      - release-*
+      - preview-*
+
+  pull_request:
+    branches: [ master ]
+    paths:
+      - src/HealthChecks.ClickHouse/**
+      - test/HealthChecks.ClickHouse.Tests/**
+      - test/_SHARED/**
+      - .github/workflows/healthchecks_clickhouse_ci.yml
+      - Directory.Build.props
+      - Directory.Build.targets
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    services:
+      clickhouse:
+        image: clickhouse/clickhouse-server:24-alpine
+        ports:
+          - 8123:8123
+        env:
+          CLICKHOUSE_DB: default
+          CLICKHOUSE_USER: default
+          CLICKHOUSE_PASSWORD: "Password12!"
+          CLICKHOUSE_DEFAULT_ACCESS_MANAGEMENT: "1"
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: |
+            6.0.x
+            7.0.x
+            8.0.x

```

We just need to adopt to https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/pull/2331 (at the moment of writing you did the right thing, we just merged a PR that forces some changes)

```suggestion
            8.0.x
            9.0.x


```